### ISSUE TITLE
Is it possible to train my own dataset?

### ISSUE THREAD

**Issue by Stephenfang51** [hiram64/ocsvm-anomaly-detection#3]
Is it possible to train my own dataset?

**Comment by hiram64** [comment]
Yes. You can use other dataset when the same format is used. I've just updated README and make master branch updated for better use. Please check it.

**Comment by zacario-li** [comment]
> Is it possible to train my own dataset?

@Stephenfang51 try this code:
[One Class SVM](https://github.com/zacario-li/OneClassSVM)

**Comment by Wangjyddddddd** [comment]
> > Is it possible to train my own dataset?
> 
> @Stephenfang51 try this code: [One Class SVM](https://github.com/zacario-li/OneClassSVM)

请问这个有对应的参考文章吗？

**Comment by zacario-li** [comment]
> > > Is it possible to train my own dataset?
> > 
> > 
> > @Stephenfang51 try this code: [One Class SVM](https://github.com/zacario-li/OneClassSVM)
> 
> 请问这个有对应的参考文章吗？

Low-Altitude Aerial Video Surveillance via One-Class SVM Anomaly Detection from Textural Features in UAV Images
